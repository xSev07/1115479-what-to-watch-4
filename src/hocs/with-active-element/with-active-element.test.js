import React from "react";
import renderer from "react-test-renderer";
import withActiveElement from "./with-active-element";

const Component = () => {
  return (<></>);
};

const WrappedComponent = withActiveElement(Component);

it(`Should withActiveElement render correctly`, () => {
  const tree = renderer
    .create(
        <WrappedComponent
          elements={[`elem1`, `elem2`, `elem3`]}
        />
    ).toJSON();

  expect(tree).toMatchSnapshot();
});
