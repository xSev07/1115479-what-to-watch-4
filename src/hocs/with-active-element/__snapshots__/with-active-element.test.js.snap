// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should withActiveElement render correctly 1`] = `null`;
